frontend/pages/SoftwarePage/components/PackageAdvancedOptions/PackageAdvancedOptions.tsx

```diff
@@ -30,17 +30,33 @@ const PKG_TYPE_TO_ID_TEXT = {
   exe: "software name",
 } as const;
 
-const getInstallHelpText = (pkgType: PackageType) => (
-  <>
-    Use the $INSTALLER_PATH variable to point to the installer.{" "}
-    {getSupportedScriptTypeText(pkgType)}{" "}
-    <CustomLink
-      url={`${LEARN_MORE_ABOUT_BASE_LINK}/install-scripts`}
-      text="Learn more about install scripts"
-      newTab
-    />
-  </>
-);
+const getInstallHelpText = (pkgType: PackageType) => {
+  if (pkgType === "exe") {
+    return (
+      <>
+        For Windows, Fleet only creates install scripts for .msi packages. Use the $INSTALLER_PATH variable to point to the installer. {" "}
+        {getSupportedScriptTypeText(pkgType)}.{" "}
+        <CustomLink
+          url={`${LEARN_MORE_ABOUT_BASE_LINK}/exe-install-scripts`}
+          text="Learn more"
+          newTab
+        />{" "}
```

Comment:
```suggestion
        />
```

We don't need this right?